Upgrade lb explorer

jannyHou opened the pull request:
WIP
Merge similar examples (`loopback-example-relations` and `loopback-example-embedded-relations`)
Upgrade the new project to be compatible with `loopback-component-explorer`
Test cases and README.md TBD
Thanks!
created as a solution for this issue: https://github.com/strongloop/loopback-example-relations/issues/17

jannyHou commented:
hey @superkhau , please review it.
Sorry I have no idea why jenkins failed but the test runs well on my laptop. 
The code and test are ready to be reviewed, and I am looking for some ideas about the `doc` and `display` part for new functions I merged in(I mean those functions from `loopback-embedded-relations-example`). 
The embedded-relations example doesn't have details in README.md and there is no page displayed in front end, everything is checked in explorer. Should I create them all in this repo?
Thanks a lot!

superkhau commented on LICENSE:
```diff
@@ -9,13 +9,14 @@ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-THE SOFTWARE.
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
```
Remove empty line.

superkhau commented:
@jannyHou It's not ready yet, you will need to update the README.md to correspond to each step you changed. This means for me to merge this, I want to follow the tutorial in the README and end up with the same app as you and know WHY etc.

jannyHou commented:
@superkhau The `readme.md` is still the original one, I will update it to keep same with the new example ASAP today. Thanks a lot.

jannyHou commented:
@superkhau Please review it, thanks a lot.
I updated `README.md` file to explain how to create & run the app from scratch. Now user can follow the doc to build a same app as this example.
And I bring the endpoints from `loopback-example-embedded-relations` to be shown on front-end.
Related test cases are also added.
